check.go

```diff
@@ -101,7 +101,11 @@ func (c SyntheticsModelCustom) fire() {
 	}
 
 	if c.Proto == "http" {
```

Comment:
Let's create  integer constants for "http". We can take care of this in the code refactor.